pkg/controller.v1/mpi/mpijob_controller.go

```diff
@@ -1365,9 +1364,10 @@ func newLauncherRole(mpiJob *kubeflowv1.MPIJob, workerReplicas int32) *rbacv1.Ro
 // handleObject can discover the MPIJob resource that 'owns' it.
 func newLauncherRoleBinding(mpiJob *kubeflowv1.MPIJob) *rbacv1.RoleBinding {
 	launcherName := mpiJob.Name + launcherSuffix
+	saName := mpiJob.Name + launcherSuffix
```

Comment:
Should we just assign saName to launcherName, to keep it simple to read ?
```suggestion
	launcherName := mpiJob.Name + launcherSuffix
	saName := launcherName
```